builtin-functions/kphp-light/stdlib/time-functions.txt

```diff
@@ -2,17 +2,24 @@
 
 // ===== SUPPORTED =====
 
-function mktime ($h ::: int = PHP_INT_MIN, $m ::: int = PHP_INT_MIN, $s ::: int = PHP_INT_MIN,
-                $month ::: int = PHP_INT_MIN, $day ::: int = PHP_INT_MIN, $year ::: int = PHP_INT_MIN) ::: int;
+function mktime (
+    $h ::: ?int = null, $m ::: ?int = null, $s ::: ?int = null, $month ::: ?int = null, $day ::: ?int = null, $year ::: ?int = null
+) ::: int;
```

Comment:
why is there a line break for arguments here?

Reply:
In my opinion, this looks more readable (too many arguments)